src/components-examples/material/chips/chips-harness/chips-harness-example.html

```diff
@@ -1,7 +1,7 @@
-<mat-chip-list
+<mat-chip-listbox
     [disabled]="isDisabled"
     [aria-orientation]="'horizontal'">
-  <mat-chip (removed)="remove()">Chip 1</mat-chip>
-  <mat-chip (removed)="remove()">Chip 2 <span matChipRemove>remove_icon</span></mat-chip>
-  <mat-chip (removed)="remove()"><mat-chip-avatar>C</mat-chip-avatar>Chip 4</mat-chip>
-</mat-chip-list>
+  <mat-chip-option (removed)="remove()">Chip 1</mat-chip-option>
+  <mat-chip-option (removed)="remove()">Chip 2 <span matChipRemove>remove_icon</span></mat-chip-option>
```

Comment:
I know that it was there before, but shouldn't the `matChipRemove` be a `mat-icon` instead of a `span`?

Reply:
Good catch, this be a `<button/>` element and align with the best practices we document for matChipRemove (documented in chips.md)